### PR TITLE
Added YARP drivers

### PR DESCRIPTION
**Pull request by francesco-romano**

### CODE REVIEW

**Comment by francesco-romano** [comment]
PR opened to start travis build

File: cmake/FindXsensXME.cmake [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	endif()
+	
+	
+    find_path(Xsens_INCLUDE_DIR xme.h
+              HINTS "$ENV{XsensXME_ROOT}/${XME_DIR_PATH}/include")
+	
+    find_library(Xsens_xme_LIBRARY "xme${XME_ARCH}"
+                 HINTS "$ENV{XsensXME_ROOT}/${XME_DIR_PATH}/lib")
+				 
+	find_library(Xsens_xstypes_LIBRARY "xstypes${XME_ARCH}"
```
Comment (traversaro) [inline cmake/FindXsensXME.cmake]: Tabs!

Comment (francesco-romano) [reply cmake/FindXsensXME.cmake]: I love Notepad++

File: cmake/FindXsensXME.cmake [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                          INTERFACE_INCLUDE_DIRECTORIES "${Xsens_INCLUDE_DIR}"
+                          IMPORTED_LINK_INTERFACE_LANGUAGES "C"
+                          IMPORTED_LOCATION "${Xsens_xme_LIBRARY}"
+                          INTERFACE_LINK_LIBRARIES "${Xsens_xstypes_LIBRARY}")
+
+    set(XsensXME_LIBRARIES Xsens::XME)
+    set(XsensXME_INCLUDE_DIRS "${Xsens_INCLUDE_DIR}")
+  endif()
+else()
+  message(WARNING "At the moment, FindXsensXME supports only Windows.")
```
Comment (traversaro) [inline cmake/FindXsensXME.cmake]: A warning is a bit invasive (it pops up when running ccmake) when we should in the end just return that XsensXME was not found).

File: CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+include(YarpPlugin)
+include(YarpInstallationHelpers)
+set(YARP_FORCE_DYNAMIC_PLUGINS TRUE)
+# I want to install plugins in PREFIX/yarp.
+#yarp_configure_external_installation(xsens-mvn-driver WITH_PLUGINS)
+
+# Compile the plugins by default
+set(COMPILE_BY_DEFAULT ON)
+
+set(CMAKE_POSITION_INDIPENDENT_CODE ON)
```
Comment (traversaro) [inline CMakeLists.txt]: The option is CMAKE_POSITION_INDEPENDENT_CODE .

File: cmake/FindXsensXME.cmake (hunk 2 above)
Comment (francesco-romano) [reply cmake/FindXsensXME.cmake]: will remove

File: CMakeLists.txt (hunk 3 above)
Comment (francesco-romano) [reply CMakeLists.txt]: ahah the nice thing is that I looked for the documentation with the correct term.. but I wrote the wrong one in the cmake 🤣

**Comment by francesco-romano** [comment]
TODO before merging:
- [ ] remove notifications from CI :) 
- [ ] Some basic testing
- [ ] check why remote does not work
- [ ] remove debug prints

**Comment by francesco-romano** [comment]
Closing as I will open a new one from my fork

**Pull request closed without merging**